### PR TITLE
Feature/UI part2 191

### PR DESCRIPTION
**Pull request by peterluo3131**
add splash screen with risingbrain logo in readme

### CODE REVIEW

File: .DS_Store
Comment (stackdev37) [inline .DS_Store:1]: Remove this unusable file.

**Review by stackdev37 — CHANGES_REQUESTED** [review]
There are few things to be updated.

**Comment by peterluo3131** [comment]
I've updated it.

File: WatchApp/.DS_Store
Comment (stackdev37) [inline WatchApp/.DS_Store:1]: remove this file.

**Review by stackdev37 — CHANGES_REQUESTED** [review]
Please update

**Pull request merged**